stock_quantity_history_location/wizards/stock_quantity_history.py

```diff
@@ -15,6 +18,8 @@ class StockQuantityHistory(models.TransientModel):
     def open_at_date(self):
         action = super(StockQuantityHistory, self).open_at_date()
         ctx = action["context"]
+        if isinstance(ctx, str):
+            ctx = ast.literal_eval(ctx)
```

Comment:
Why are you doing this? Please use `safe_eval` if needed, but this PR message doesn't correspond to this.